1001-2000/LIT1750LahaMa.xml

```diff
@@ -31,8 +31,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             </availability>
          </publicationStmt>
          <sourceDesc>
```

Comment:
Here as well: do not delete these statements.

Reply:
OK, done.